src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -62,34 +63,40 @@
   
   <PropertyGroup Condition="'$(StringResourcesPath)'!=''">
       <CompileDependsOn>
+          AssemblyNameNormalization;
           GenerateResourcesSource;
           $(CompileDependsOn);
       </CompileDependsOn>
   </PropertyGroup>
 
-  <Target Name="GenerateResourcesSource"
-          Condition="'$(StringResourcesPath)'!='' AND '$(OmitResources)'!='true'"
-          Inputs="$(StringResourcesPath)"
-          Outputs="$(IntermediateResOutputFileFullPath)">
-
-    <GenerateResourcesCode
-        ResxFilePath="$(StringResourcesPath)" 
-        OutputSourceFilePath="$(IntermediateResOutputFileFullPath)" 
+  <Target Name="AssemblyNameNormalization" Condition="'$(StringResourcesPath)'!='' AND '$(OmitResources)'!='true'">
```

Comment:
Naming is bad.  Can you follow the same pattern we use elsewhere: name the target after the one task it calls?